### ISSUE TITLE
Kernel: Type mismatch in map_ebda in BIOS.cpp

### ISSUE THREAD

**Issue by Lubrsi** [SerenityOS/serenity#5501]
Happens during boot.
Trace:
```
[#0 Kernel]: Early access to ACPI tables for interrupt setup
[#0 Kernel]: KUBSAN: type mismatch, 'short unsigned int' (1-bit)
[#0 Kernel]: KUBSAN: at .././Kernel/VM/TypedMapping.h, line 41, column: 33
[#0 Kernel]: 0xc0486161 (next: 0xc0107fb8)
[#0 Kernel]: 0xc013604d (next: 0xc0108038)
[#0 Kernel]: 0xc058f456 (next: 0xc01080a8)
[#0 Kernel]: 0xc05aa9be (next: 0xc0108108)
[#0 Kernel]: 0xc05aad99 (next: 0xc0108158)
[#0 Kernel]: 0xc05aae5a (next: 0xc01081b8)
[#0 Kernel]: 0xc05cfde4 (next: 0xc0108298)
[#0 Kernel]: 0xc0100145 (next: 0x00000000)
[#0 Kernel]: KUBSAN: type mismatch, 'short unsigned int' (1-bit)
[#0 Kernel]: KUBSAN: at ../Kernel/Arch/PC/BIOS.cpp, line 49, column: 24
[#0 Kernel]: 0xc0486161 (next: 0xc0107fb8)
[#0 Kernel]: 0xc013606b (next: 0xc0108038)
[#0 Kernel]: 0xc058f456 (next: 0xc01080a8)
[#0 Kernel]: 0xc05aa9be (next: 0xc0108108)
[#0 Kernel]: 0xc05aad99 (next: 0xc0108158)
[#0 Kernel]: 0xc05aae5a (next: 0xc01081b8)
[#0 Kernel]: 0xc05cfde4 (next: 0xc0108298)
[#0 Kernel]: 0xc0100145 (next: 0x00000000)
```

**Comment by Hendiadyoin1** [comment]
At this point there only seems to be a alignment mismatch at this address, can you confirm?

**Comment by Hendiadyoin1** [comment]
I can only Find
```
[#0 Kernel]: KUBSAN: reference binding to misaligned address 0xc1104413 of type 'short unsigned int'
[#0 Kernel]: KUBSAN: at .././Kernel/VM/TypedMapping.h, line 41, column: 33
[...]
[#0 Kernel]: KUBSAN: load of misaligned address 0xc1104413 of type 'short unsigned int'
[#0 Kernel]: KUBSAN: at ../Kernel/Arch/PC/BIOS.cpp, line 49, column: 24
```
in my logs, the type mismatch seems to be resolved

**Comment by boricj** [comment]
Fixed by #5674.
